app/grandchallenge/hanging_protocols/models.py

```diff
@@ -0,0 +1,102 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from guardian.shortcuts import assign_perm
+
+from grandchallenge.core.models import TitleSlugDescriptionModel, UUIDModel
+from grandchallenge.core.validators import JSONValidator
+
+
+class ImagePort(models.TextChoices):
+    MAIN = "M", "Main"
+    SECONDARY = "S", "Secondary"

```

Comment:
Do you know why these first 2 have a single character choice while the others are fully typed out? I see you moved this from the ReaderStudy model to here and it was already like this, but I'm just curious why it's like this.